js/src/lib/components/PipelineIcon/PipelineIcon.svelte

```diff
@@ -59,6 +60,7 @@
 		"fill-mask": IconFillMask,
 		"sentence-similarity": IconSentenceSimilarity,
 		"text-to-speech": IconTextToSpeech,
+		"text-to-audio": IconTextToAudio,
```

Comment:
if it is exact same icon as `IconTextToSpeech`, why not just use `IconTextToSpeech` ?

Reply:
Good call: I fixed it in the latest commit.